src/python/ensembl/production/datafile/scraper/__init__.py

```diff
@@ -0,0 +1,12 @@
+#  See the NOTICE file distributed with this work for additional information
+#  regarding copyright ownership.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""DataFile Scraper package. Used to fetch file metadata for Ensembl flat files"""
```

Comment:
Is the new line not expecte3d here?

Reply:
AFAIK, module docstring should be at the top of the module, and that's about it...

Reply:
No problem, I said that because of one comment about it that Jorge had on my PR en ensembl-py